### ISSUE TITLE
model.RunModel hangs when model is 'Optimized'

### ISSUE THREAD

**Issue by carljones4** [Microsoft/vs-tools-for-ai#52]
When a Tensorflow model .pb file is imported using the Visual Studio Tools for AI import wizard, if the "Optimize" checkbox is checked, when executed, the call to manager.RunModel will run, but never finish.

My example is a very simple network built with Keras...
    Conv2D(64, kernel_size=3, activation='relu', input_shape=input_shape),
    Dropout(0.2),
    Conv2D(32, kernel_size=3, activation='relu'),
    Dropout(0.2),
    Flatten(),
    Dense(num_classes, activation='softmax')])

Running ONNX 1.3, Tensorflow 1.10.0.
